landbosse/excelio/XlsxManagerRunner.py

```diff
@@ -107,3 +117,70 @@ def extract_details_lists(self, runs_dict):
                 if key.endswith('_csv'):
                     runs_for_csv.extend(value)
         return runs_for_csv
+
+    def read_project_and_parametric_list_from_xlsx(self):
+        """
+        This method reads both the project and parametric list from the
+        project_list xlsx. It returns them as a tuple.
+
+        To ensure backward compatibility with project list spreadsheets
+        that contain only one sheet, the following actions take place
+
+        1. Project list spreadsheet contains one sheet: The only sheet
+        that is present is read as the project list, and a dataframe for
+        parametrics is returned that has all the columns but no rows so
+        it can be used in joins. It doe snot matter what the sheet name
+        is.
+
+        2. Project list spreadsheet contains two sheets: The sheet named
+        "Project list" is assumed to be the project list and the sheet
+        named "Parametric list" is assumed to be the specifications for
+        the parametric variables. If one or both of those tab names are
+        missing, an exception is raised
+
```

Comment:
I don't think the back-compatibility is working correctly right now. I get the following error and I think the root of the problem is in this method. 
![image](https://user-images.githubusercontent.com/9435759/68960013-92c17c80-07cf-11ea-97fe-7492d58c9a81.png)

Reply:
I found this problem about the same time you did. I fixed it and re-ran validation using data compatible with version 2.1.5 and it works now.